src/main/java/uk/gov/hmcts/reform/preapi/controllers/RecordingController.java

```diff
@@ -84,4 +85,19 @@ public ResponseEntity<Void> deleteRecordingById(
         recordingService.deleteById(bookingId, recordingId);
         return ResponseEntity.ok().build();
     }
+
+    @PutMapping("/{recordingId}/share")
+    @Operation(operationId = "shareRecordingById", summary = "Share a Recording")
+    public ResponseEntity<Void> shareRecordingById(
+        @PathVariable UUID bookingId,
+        @PathVariable UUID recordingId,
+        @RequestBody ShareRecordingDTO shareRecordingDTO
+    ) {
+        // TODO Ensure user has access to share the recording
+        if (!recordingId.equals(shareRecordingDTO.getCaptureSessionId())) {
```

Comment:
Shouldn't the recording id relate to a recording. Then you could get the capture session related to the recording.

Reply:
So you would do something like:
1.  Get the recording by the recording id
2. Get the related capture session's id for the recording
3. Check the id found matches the id in the dto.

Reply:
Sharing a booking now instead